pkg/controller/performanceprofile/performanceprofile_controller.go

```diff
@@ -316,7 +316,27 @@ func (r *ReconcilePerformanceProfile) Reconcile(request reconcile.Request) (reco
 	return reconcile.Result{}, nil
 }
 
+func isPaused(profile *performancev1alpha1.PerformanceProfile) bool {
```

Comment:
in case this is merged after #40: this would be a good fit for the new `pkg/controller/performanceprofile/components/profile/profile.go`

Reply:
yup, fixed. I added the function to that pkg, but had to import the profile package with the `profileutils` alias to prevent conflicting with the commonly named `profile` struct that represents the performance profile.